### DISCUSSION TITLE
创建计算组后无法验证正确性

### DISCUSSION THREAD

**Discussion by coulsonm** [ByConity/ByConity#649] [Q&A] [answered]
物理模式部署，使用计算组遇到三个问题：
1.文档中提到计算组/计算组副本/物理计算组/共享计算组等概念，实际使用中只能指定到计算组这一层
需要在query中指定 SETTINGS virtual_warehouse = "vwname"

2.指定计算组步骤比较麻烦和固化，需要先修改read worker的配置文件，指定计算组和计算组副本，然后在client端再创建同名计算组和计算组副本才能使用

3.指定计算组运行后，无法判断query运行在哪个节点上，即无法验证计算组的有效性

**Comment by dmthuc** [comment, marked as answer]
Hi @coulsonm , thank you for asking the question. I would like to answer question like follow
1. we can also specify which vw is using by set it in table setting. And if you don't specify then it will use the `default` vw for read and write for the remain. 
2. You only need to edit the config file to set a worker to a worker group. Don't need to do any other thing. I'm not sure where do you read that you need to do other stuff
3. You can find which vw is use by select system.query_log

Let me know if you have any other question
